iotilecore/RELEASE.md

```diff
@@ -2,8 +2,9 @@
 
 All major changes in each released version of `iotile-core` are listed here.
 
-## HEAD
+## 5.0.6
```

Comment:
5.0.6 is already released -- you might need to rebase this branch with master https://pypi.org/project/iotile-core/#history

Reply:
Thanks for that. Just rebased.

Reply:
Sorry, I commented a touch too soon. I was just working on merging Dima's changes to master, so you may have to do it one more time. It shouldn't touch any of the same files, at least, because I coudn't update RELEASE.md with the notes that it should actually have..